doc/nrf/security/tfm.rst

```diff
@@ -254,38 +267,106 @@ We will decrease the size of the (optional) ``mcuboot_pad`` partition and thus t
       address: 0x8000
       size: 0x4000
 
+Analyzing TF-M partition size
```

Comment:
Some things that are missing:
-  ITS + PS partition size 
-  tfm_ram_report, tfm_rom_report